includes/db-updates.php

```diff
@@ -1,5 +1,22 @@
 <?php
 /**
+ * Version 3.0.8
+ *
+ * Force update for older versions to call \dbdelta in install() method to fix column widths.
+ *
+ * @param string $db_version
+ * @param string $current_version
+ *
+ * @return string
+ */
+function wp_stream_update_308( $db_version, $current_version ) {
```

Comment:
Since this is a small change, and won't take long to execute, we should make it an auto update (so the user doesn't need to press a button to update the tables). If you agree, the function name just needs to change to `wp_stream_auto_update_308`.